docs/modules/plugins/pages/plugin-cross-browser-testing.adoc

```diff
@@ -0,0 +1,51 @@
+:grid-default-hostname: hub.crossbrowsertesting.com
+
+= CrossBrowserTesting Plugin
+
+The plugin provides integration with continuous testing cloud https://crossbrowsertesting.com/[CrossBrowserTesting].
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-crossbrowsertesting', version: '{current-version}')
+----
+
+== Profiles
+
+[cols="1,2"]
+|===
+
+|`crossbrowsertesting/web`
+|Use in conjunction with web application profiles.
+
+|===
+
+CrossBrowserTesting profiles can be enabled by adding the corresponding entry to `configuration.profiles` property.
+
+.Configure Chrome on CrossBrowserTesting
+[source,properties,subs="attributes+"]
+----
+configuration.profiles=crossbrowsertesting/web,web/desktop/chrome
+----
+
+[IMPORTANT]
+CrossBrowserTesting profile must be first.
+
+Explore https://support.smartbear.com/crossbrowsertesting/docs/automated-testing/automation-capabilities.html/[Automation Capabilities] for more fine-grained CrossBrowserTesting configuration.
+
+TIP: Links to CrossBrowserTesting test runs can be found in Allure report
```

Comment:
it looks like a lot of these content is copy-pasted, doesn't it?

Reply:
:)

Reply:
it's just a coincidence :)